core/src/main/java/edu/stanford/futuredata/macrobase/pipeline/BasicBatchPipeline.java

```diff
@@ -2,17 +2,19 @@
 
 import edu.stanford.futuredata.macrobase.analysis.classify.Classifier;
 import edu.stanford.futuredata.macrobase.analysis.classify.PercentileClassifier;
+import edu.stanford.futuredata.macrobase.analysis.classify.RawThresholdClassifier;
```

Comment:
Minor nit, but I would rename ArithmeticClassifier to something like StdDevGroupClassifier